### DISCUSSION TITLE
刷活动图时出现错误，error文件夹里对应的文件已添加，错误页面截图已添加

### DISCUSSION THREAD

**Discussion by nneeiill2508** [LmeSzinc/AzurLaneAutoScript#1357] [Q&A] [answered]
文件夹名称：1658075826004
![2022-07-18_00-37-05-403276](https://user-images.githubusercontent.com/71597704/179416214-5878357d-db60-491a-9a42-d204caf66748.png)
[log.txt](https://github.com/LmeSzinc/AzurLaneAutoScript/files/9128150/log.txt)
![alas错误截图](https://user-images.githubusercontent.com/71597704/179416282-5d29c574-7169-4f47-985c-7e56302eac93.png)

**Comment by LmeSzinc** [comment, marked as answer]
活动通用设置，活动 PT 限制，只能填数字 `32250`，不可以使用千分位 `32,250`

**Reply by nneeiill2508** [reply]
谢谢
